tests/components/tessie/test_update.py

```diff
@@ -0,0 +1,17 @@
+"""Test the Tessie update platform."""
```

Comment:
Can you maybe add a build number to the fixture and test if this is split correctly?

Reply:
There is a build in the fixture already.
https://github.com/home-assistant/core/blob/dev/tests/components/tessie/fixtures/vehicles.json#L204

Reply:
Oh I was looking at the 2 changed lines in the fixture

Reply:
Yeah that does not have a build, hence why I wanted to normalize it.